src/sentry/search/snuba/backend.py

```diff
@@ -233,9 +235,13 @@ def _query(self, project, retention_window_start, group_queryset, tags, environm
             # so this entire search was against a time range that is outside of
             # retention. We'll return empty results to maintain backwards compatability
             # with Django search (for now).
-            return Paginator(Group.objects.none()).get_result()
+            return EMPTY_RESULT
 
-        assert start < end
+        if start > end:
```

Comment:
`>=` ?

Reply:
It's really, really minor but I figured it was legal to search for events within a single second?

Reply:
Hmm, yeah good point, snuba checks that `from_date <= to_date` so that is legal, but it also uses a half-open interval on timestamp so practically speaking you would get nothing back. 

```
    ('timestamp', '>=', from_date),
    ('timestamp', '<', to_date),
```

Reply:
Oh, whoops, OK `>=` it is in that case.